german/the-black-hack.md

```diff
@@ -182,23 +183,23 @@ Charaktere können Trefferpunkte durch Zaubersprüche, Heiltränke und Fähigkei
 |    3   | **Knochenbrüche** - STÄ, GES und AUS zeitweilig -2 für 1 Tag. |
 |    4   | **Verkrüppelt** - STÄ oder GES wird permanent um 2 gesenkt.   |
 |    5   | **Entstellt** - CHA fällt auf 4.                              |
-|    6   | **Tod** - Nicht mehr am Leben.                                |
+|    6   | **Tot** - Nicht mehr am Leben.                                |
```

Comment:
Hmm, von der Reihe her müsste da natürlich "Tot" stehen, das, was jemand ist. Andererseits finde ich es in Ordnung, wenn "Der Tod" sich unterscheidet, da er ja eine eigene Kategorie darstellt...